viewer/viewer.js

```diff
@@ -17,7 +17,7 @@
  */
 'use strict';
 
-const MIN_DB_VERSION = 70;
+const MIN_DB_VERSION = 72;
```

Comment:
we need to upgrade the version or else the history index will complain about the esquery field

Reply:
leave it to 70 for now